src/memote/checks/Annotation.jl

```diff
@@ -19,31 +19,31 @@ is_annotated_gene(model, gid) = !isempty(gene_annotations(model, gid))
 """
 $(TYPEDSIGNATURES)
 
-Helper function to find all the annotations that are missing for a component in a model.
+Helper function to find all the annotations for a component in a model.
 """
-function _find_unannotated_components(
+function _find_annotated_components(
```

Comment:
why underscore? (this is not being exported right?)

Reply:
the leading underscore was pre-existing, I can remove it

Reply:
yeah it is not exported

Reply:
yeah my point is that now with better modularization the underscores are unnecessary. Feel free to remove later as we pass along this